.github/workflows/pr-title-check.yml

```
@@ -0,0 +1,22 @@
+name: PR Title Check
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  pr-title:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Validate PR Title
+        run: |
+          if [[ ! "${{ github.event.pull_request.title }}" =~ ^(feat|fix|chore|major)\([a-zA-Z0-9_-]+\):\ .+$ ]]; then
```

We also need to add another convention to the title

```suggestion
          if [[ ! "${{ github.event.pull_request.title }}" =~ ^(feat|fix|chore|major|docs|ci)\([a-zA-Z0-9_-]+\):\ .+$ ]]; then
```

---

https://github.com/juntossomosmais/atomium/pull/526/commits/d02ccf62710557500f7bc0b65ea588605b7f7ce7